framework/src/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -125,6 +125,9 @@
     /** Should use flow analysis? */
     protected boolean useFlow;
 
+    /** Is dataflow ever used for this checker? **/
```

Comment:
I assume the following: we disable useFlow in certain situations and everUseDataFlow is the initial value of useFlow. Is that right?
Should the field be more symmetric to useFlow? Or is "DataFlow" something else?
The temporal dependency isn't clear from the name, but that is more of a problem with the lack of description of when "useFlow" is changed.